jsk_perception/node_scripts/ofa_client_node.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+import actionlib
+import base64
+import cv2
+from cv_bridge import CvBridge
+from dynamic_reconfigure.server import Server
+from jsk_perception.cfg import VQAQueryConfig
+from jsk_recognition_msgs.msg import QuestionAndAnswerText
+from jsk_recognition_msgs.msg import VQATaskAction
+from jsk_recognition_msgs.msg import VQATaskFeedback
+from jsk_recognition_msgs.msg import VQATaskResult
+from jsk_recognition_msgs.msg import VQAResult
+import json
+import requests
+from requests.exceptions import ConnectionError
+import rospy
+from sensor_msgs.msg import Image
+from std_msgs.msg import String
+
+def ros_img_to_base(ros_img, bridge):
+    cv_img = bridge.imgmsg_to_cv2(ros_img, desired_encoding="bgr8")
+    # convert to base64
+    encimg = cv2.imencode(".png", cv_img)[1]
+    img_str = encimg.tostring()
+    img_byte = base64.b64encode(img_str).decode("utf-8")
+    return img_byte
+
+class OFAClientNode(object):
+    def __init__(self):
+        self._bridge = CvBridge()
+        # inference server configuration
+        self.host = rospy.get_param("~host", default="localhost")
+        self.port = rospy.get_param("~port", default=8888)
+        # Caption
+        self.caption_image_sub = rospy.Subscriber("~caption_image", Image,
```

Comment:
Set ` self.caption_queries = None` before the subscriber? Same with `vqa_queries`.
```
Traceback (most recent call last):                                                          [231/622]
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback 
    cb(msg)                                                                                          
  File "/home/affonso/sandbox/recognition_ws/src/jsk_recognition/jsk_perception/node_scripts/ofa_clie
nt_node.py", line 69, in caption_topic_cb                                                            
    if not self.caption_queries:                                                                     
AttributeError: 'OFAClientNode' object has no attribute 'caption_queries'  
```